### PR TITLE
Add sbt-release

### PR DESCRIPTION
**Pull request by joelsonoda**
* Add sbt-release plugin
* Set the correct version number in version.sbt

### CODE REVIEW

**Comment by joelsonoda** [comment]
Tested releasing to my own internal maven repo using sbt release...

**Comment by graingert** [comment]
@joelsonoda I'm generally doing this via bintray

**Comment by joelsonoda** [comment]
@graingert I think this will work with bintray as it does with my corp archiva instance...it for the most part just manages setting version numbering rather than having to edit files.

**Comment by joelsonoda** [comment]
If you don't want to use sbt-release to do the version numbering/tagging, that's ok with me. Will you delete the branch in that case?

File: project/plugin-releases.sbt [hunk 1]
```diff
@@ -1,3 +1,6 @@
 addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.1")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
+
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.3")
+
```
Comment (graingert) [inline project/plugin-releases.sbt]: spare whitespace

**Comment by graingert** [comment]
I'm up for it to manage the version numbers etc.

**Pull request merged**